web/api/authentication.thrift

```diff
@@ -69,6 +73,18 @@ service codeCheckerAuthentication {
                       2: string authString)
                       throws (1: codechecker_api_shared.RequestFailed requestError),
 
+  // Handles reguesting state stored in database for OAuth
+  bool validateOAuthSession(1: string state),
+
+  // Handles reguesting access_token stored in database for OAuth verification
+  bool validateOAuthTokenSession(1: string access_code)
```

Comment:
Remove this from the public API! This is a huge security vulnerability! This way, an unauthenticated user can scan for valid oauth access tokens!

Move this function to be server internal and use it from the test.